### ISSUE TITLE
Start worker without kafka supervisor

### ISSUE THREAD

**Issue by steffenix** [kafkaex/kafka_ex#201]
Using the example provided on the readme
```
GenServer.start_link(KafkaEx.Server,
  [
    [uris: Application.get_env(:kafka_ex, :brokers),
     consumer_group: Application.get_env(:kafka_ex, :consumer_group)],
    :no_name
  ]
)
```

I get the error:

``` 
(UndefinedFunctionError) function KafkaEx.Server.init/1 is undefined or private

(kafka_ex) KafkaEx.Server.init([[uris: [{"10.10.101.1", 9092}, {"10.10.101.2", 9092}, {"10.10.101.3", 9092}], consumer_group: "kafka_ex_messenger"], :no_name]) 
```

I am using the latest version of kafka_ex 0.6.5.

----

After investigating, I can start it using KafkaEx.Server0P8P2 instead of KafkaEx.Server.

**Comment by dantswain** [comment]
Thanks, @steffenix !

**Comment by joshuawscott** [comment]
Seems to be fixed by #204
